class-00/lab-b/starter-code/app.js

```diff
@@ -190,17 +163,18 @@ Student.prototype.scope = function() {
 };
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scope());
+console.log('joe scope', joe.scope());
 
-Student.prototype.scopeArrow = () => console.log(this);
+Student.prototype.scopeArrow = () => console.log('this', this);
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scopeArrow());
+console.log('arrow', joe.scopeArrow());
 
 // TODO: Write a COMMENT below to answer the following questions.
 // 1. What is "this" when joe.scope() is invoked?
-//
+//  this is undefined.

```

Comment:
The 'this' should still be defined. What you probably noticed is the return value of console.log(), which is undefined. Above the 'undefined' return value you should see the output log that you had requested.